types/types.go

```diff
@@ -32,6 +32,8 @@ const (
 	CRDAPIVersionV1beta1  = "longhorn.io/v1beta1"
 	CRDAPIVersionV1beta2  = "longhorn.io/v1beta2"
 	CurrentCRDAPIVersion  = CRDAPIVersionV1beta2
+
+	CurrentLonghornVersion = "v1.3.0"
```

Comment:
Is it possible to read the version value in file `longhorn-manager/version` and then set to `CurrentLonghornVersion` during compilation? Then, we don't have to maintain two `version` values.

Reply:
Do you mean adding an ENV in the YAML file `deploy/install/02-components/01-manager.yaml`?
BTW, what is another version value? the image tag?

Reply:
I'm just thinking if we can reuse the value in https://github.com/longhorn/longhorn-manager/blob/master/version .

Reply:
Oh, I never realize there is such a file in repo `longhorn-manager`... Where do we use it?

Reply:
Hmm, I have no idea.
@innobead Do you know the usage of the `version` file?

Reply:
https://github.com/longhorn/longhorn-manager/blob/c09016aafa9e7f582ed2f7ecafa2d823795cd353/scripts/build#L9

Reply:
using a meta package like in the engine is a good approach :)

Reply:
The version value used in [the build script you mentioned](https://github.com/longhorn/longhorn-manager/blob/c09016aafa9e7f582ed2f7ecafa2d823795cd353/scripts/build#L9) is from the [version script](https://github.com/longhorn/longhorn-manager/blob/840ae441597ac6af0904946db7d40916d959f9f8/scripts/version#L11-L150, which is actually using git commit or tag rather than the file we discussed. Should we improve this part? And how? Since right now once there is no git tag when building one longhorn manager image, the version number will be the commit ID. Should we differentiate the git commit ID from the version number and always use the value in https://github.com/longhorn/longhorn-manager/blob/master/version for this upgrade path refactoring?